Packs/qualys/Scripts/QualysCreateIncidentFromReport/QualysCreateIncidentFromReport.yml

```diff
@@ -0,0 +1,40 @@
+commonfields:
+  id: QualysCreateIncidentFromReport
+  version: -1
+name: QualysCreateIncidentFromReport
+script: ''
+type: python
+subtype: python2
+tags:
+- qualys
+comment: |-
+  Create incidents from a Qualys report (XML), based on the Qualys asset ID and vulnerability ID (QID).
+  Duplicate incidents are not created for the same asset ID and QID.
+system: true
+args:
+- name: entryID
+  required: true
+  description: War room entryID of the XML report.
+- name: maxFileSize
+  description: Maximum file size to load, in bytes. Default is 1024 KB.
+- name: minSeverity
+  auto: PREDEFINED
+  predefined:
+  - '1'
+  - '2'
+  - '3'
+  - '4'
+  - '5'
+  description: The minimum Qualys severity to create incidents for.
+- name: incidentType
+  description: The incident type to create incidents for. Default is "Vulnerability".
+scripttarget: 0
+runas: DBotWeakRole
+runonce: false
+dependson:
+  must:
+  - qualys-host-list
+tests:
+- No test
+fromversion: 5.0.0
+dockerimage: demisto/python:2.7.18.27799
```

Comment:
let's update to the most recent py3 image

Reply:
this isnt a new script, does this still make sense to do? I think it would be best leaving it at python2 to make the smallest changes possible here

Reply:
updated image